app/platform/forms.py

```diff
@@ -5,5 +5,10 @@
 
 
 class GrantForm(FlaskForm):
-    name = StringField("Grant name", validators=[DataRequired()], widget=GovTextInput())
+    name = StringField(
+        "Grant name",
+        validators=[DataRequired("Enter a grant name")],
+        filters=[lambda x: x.strip() if x else x],
+        widget=GovTextInput(),
+    )
     submit = SubmitField("Submit", widget=GovSubmitInput())
```

Comment:
```suggestion
    submit = SubmitField("Set up grant", widget=GovSubmitInput())
```

Reply:
this form gets used in set up/ edit so this is done in the template - I don't love it

Reply:
Ok, maybe just have the label here as an empty string then?

Reply:
Sounds good - as we recreate this pattern in various places I think I'd be in favour of explicit forms for creating and updating. I don't think there's going to end up being much overlap

Reply:
agree